app/views/order_items/_order_item.html.erb

```diff
@@ -5,6 +5,17 @@
         <%= image_tag product.avatar %>
       </div>
       <div class="caption">
+        <h6>
+          <% if product.calories != nil %>
```

Comment:
It's even better to switch `unless` and `else` to `if` and `else` and swap the blocks